pkg/minikube/registry/drvs/podman/podman.go

```diff
@@ -108,14 +108,21 @@ func status() registry.State {
 	ctx, cancel := context.WithTimeout(context.Background(), 6*time.Second)
 	defer cancel()
 
+	var o []byte
 	// Quickly returns an error code if service is not running
 	cmd := exec.CommandContext(ctx, oci.Podman, "version", "--format", "{{.Server.Version}}")
 	// Run with sudo on linux (local), otherwise podman-remote (as podman)
 	if runtime.GOOS == "linux" {
-		cmd = exec.CommandContext(ctx, "sudo", "-k", "-n", oci.Podman, "version", "--format", "{{.Version}}")
-		cmd.Env = append(os.Environ(), "LANG=C", "LC_ALL=C") // sudo is localized
+		// Run podman as USER, if that fails run podman as sudo
+		o, err = cmd.Output()
+		if err != nil && sudoNeedsPassword() {
```

Comment:
why we run this only if sudo needs password? 
what if sudo does NOT need password, we dont wanna run it then?